packages/terra-popup/tests/wdio/popup-spec.js

```diff
@@ -0,0 +1,16 @@
+/* global  browser,Terra */
+const viewports = Terra.viewports('tiny', 'medium');
+
+describe('Popup', () => {
+  describe('Arrow Popup', () => {
+    beforeEach(() => browser.url('/#/tests/popup-tests/arrow'));
+
+    Terra.should.beAccessible({ viewports });
+    Terra.should.matchScreenshot({ viewports });
+    Terra.should.themeEachCustomProperty({
```

Comment:
The screenshot sizes are too small for this method. Add the #root selector we talked about.